share/wake/lib/.wakeroot

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
What's the advantage of allowing this edge case?

Reply:
You can write `touch .wakeroot` (rather than `echo "{}" > .wakeroot`) when you want to create a playground directory (for example creating a single file demo example for someone).
I guess we could use `wake --init .` instead though

Reply:
Well an empty string is not valid json so we're not changing the parser.